src/pager/mod.rs

```diff
@@ -96,11 +100,17 @@ impl Pager {
 
     /// Perform setup. The setup function is run once at the start.
     fn setup(&mut self, stdout: &mut std::io::Stdout) -> std::io::Result<()> {
+        // Determine the layout sizes
         let sizes = layout::distribute(
             self.height,
             &vec![layout::Size::Flexible, layout::Size::Fixed(1)],
         );
+
+        // Setup subcomponents
         self.view.setup(stdout, (self.width, sizes[0]))?;
+        self.command_line
+            .setup((1, self.height as u16 - 1), (self.width, sizes[1]))?;
```

Comment:
The position tuple has `1` as a magic number. This should probably just be `0, self.height as u16` and any inner padding should be handled by the component.

```suggestion
            .setup((0, self.height as u16), (self.width, sizes[1]))?;
```

Maybe the `self.height as u16 - 1` was because positions are zero-indexed while height isn't? Not sure, but should check.

Reply:
Fixed in #981a6c4